libs/utils/executor.py

```diff
@@ -318,6 +318,30 @@ def _setup_sched_features(self, tc):
     def get_run_dir(target):
         return os.path.join(target.working_directory, TGT_RUN_DIR)
 
+    def _selinux_mode_unused(self):
```

Comment:
nitpicking: it's usually easier to read code with variables that are something instead of "not something", to avoid double negation